src/themes/default/components/Carousel/_carousel.scss

```diff
@@ -52,16 +51,14 @@
             @extend .slick-prev;
             right: $message-spacing-thick * -1;
             left: auto;
-            border-top-right-radius: 0px;
-            border-bottom-right-radius: 0px;
-            border-top-left-radius: $message-border-radius-small;
-            border-bottom-left-radius: $message-border-radius-small;
+            border-radius: $message-border-radius-small 0px 0px $message-border-radius-small;
```

Comment:
why did you change this?
also why did you convert it to the terse form? it was easier to read previously.